pkg/consensus/consensus.go

```diff
@@ -725,7 +725,7 @@ func (s *service) newBlockFromContext(ctx *dbft.Context) block.Block {
 	var err error
 	cfg := s.Chain.GetConfig().ProtocolConfiguration
 	if cfg.ShouldUpdateCommitteeAt(ctx.BlockIndex) {
```

Comment:
And this one as well.

Reply:
And this one is definitely not the optimisation as far.

Reply:
And here as well.